libretranslate/umbrel-app.yml

```diff
@@ -0,0 +1,35 @@
+manifestVersion: 1
+id: libretranslate
+category: ai
+name: Libretranslate
+version: "1.5.6"
+tagline: Free and Open Sourcemachine translation API
+description: >-
+  Free and Open Source Machine Translation API, entirely self-hosted.
+  Unlike other APIs, it doesn't rely on proprietary providers such as Google or Azure to perform translations.
+  Instead, its translation engine is powered by the open source Argos Translate library.
+releaseNotes: >-
+  What's Changed\:
+  ltmanage in bin by @jmlord in #580
+  Add --req-time-cost by @pierotofy in #584
+  Improve model contributing doc by @argosopentech in #585
+  Added TypeScript language binding library. by @tderflinger in #590
+  Specify the supported color schemes by @Phoenix616 in #593
+  add\: R binding by @myanesp in #594
+  fix\: make libretranslate works with ipv6 by @cyrinux in #596
+
```

Comment:
For the initial release to the app store we can just remove release notes and go with:
`releaseNotes: ""`

Reply:
Yea, you're probably right